src/SPU.cpp

```diff
@@ -190,6 +192,14 @@ void SetBias(u16 bias)
     Bias = bias;
 }
 
+void SetApplyBias(bool enable) {
```

Comment:
brackets are wrong

Reply:
yep, it should look like this
```suggestion
void SetApplyBias(bool enable)
{
```